src/cmd/keybaseca/keybaseca.go

```diff
@@ -7,7 +7,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/keybase/bot-sshca/src/keybaseca/ca"
+	"github.com/keybase/bot-sshca/src/keybaseca/bot"
```

Comment:
wait what? didn't you just change _to_ `ca` _from_ `bot`?

Reply:
sorry if this was confusing -- i changed to ca then decided to change it back to bot, after re-reading your comment (`"if i had my druthers, bot would only refer to the ca bot"`).